ci/build_linux.sh

```diff
@@ -17,6 +17,12 @@ else
     export GIT_URL=https://github.com/${GITHUB_REPOSITORY}.git
     export GIT_BRANCH=origin/$(basename ${GITHUB_REF})
     export GIT_COMMIT=${GITHUB_SHA}
+    # remove unnecssary files to save disk space on github
+    # base on the workaround provided here:
+    # https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+    sudo rm -rf /usr/share/dotnet
```

Comment:
are we sure to remove these three directory?

Reply:
This is a workaround provided by a person from github for disk space issue.
Remove those directories only when build on github hosted runners and I think it would be restored from their image each time when starting a new job.
Those directories is from github hosted runners and not used by our build, so should be safe to remove them.